apps/api/src/hyp3_api/validation.py

```diff
@@ -215,6 +215,24 @@ def check_bounding_box_size(job: dict, _, max_bounds_area: float = 4.5) -> None:
         )
 
 
+def _has_rtc_static_coverage(granule_name: str) -> bool:
+    burst_id, swath = granule_name.split('_')[1:3]
```

Comment:
See my pending question to @forrestfwilliams in Mattermost regarding the term "burst ID".

Reply:
Forrest says burst ID = burst number + swath, so we should change the variable name here to `burst_number`.